clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/deploy/converters/UpdateServiceConfigAtomicOperationConverter.java

```diff
@@ -0,0 +1,32 @@
+package com.netflix.spinnaker.clouddriver.ecs.deploy.converters;
+
+import com.netflix.spinnaker.clouddriver.ecs.EcsOperation;
+import com.netflix.spinnaker.clouddriver.ecs.deploy.description.CloneServiceDescription;
+import com.netflix.spinnaker.clouddriver.ecs.deploy.description.UpdateServiceConfigDescription;
+import com.netflix.spinnaker.clouddriver.ecs.deploy.ops.CloneServiceAtomicOperation;
+import com.netflix.spinnaker.clouddriver.ecs.deploy.ops.UpdateServiceConfigAtomicOperation;
+import com.netflix.spinnaker.clouddriver.orchestration.AtomicOperation;
+import com.netflix.spinnaker.clouddriver.orchestration.AtomicOperations;
+import com.netflix.spinnaker.clouddriver.security.AbstractAtomicOperationsCredentialsSupport;
+import org.springframework.stereotype.Component;
+
+import java.util.Map;
+
+@EcsOperation(AtomicOperations.UPDATE_LAUNCH_CONFIG)
+@Component("ecsUpdateLaunchConfig")
```

Comment:
Hi @robzienert , thanks for reviewing :)

You are kind of right.  Services launch tasks using task definitions.  However, you cannot modify a task definition - you can only make a new revision.  Therefore you need to create a new task definition revision, and then tell your service to use that new task definition revision.  

Also, some parameters would clearly fall on the service rather than the task definition.  For instance, the number of tasks to run is at the service level, while the task definition defines what resources a task may reserve

Does this make more sense now?